Framework/Algorithms/test/GetDetOffsetsMultiPeaksTest.h

```diff
@@ -299,11 +295,14 @@ class GetDetOffsetsMultiPeaksTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().addOrReplace("temp_event_ws", WS);
     WS->getAxis(0)->unit() =
         Mantid::Kernel::UnitFactory::Instance().create("dSpacing");
+
     const auto &X = WS->x(0);
     auto &Y = WS->mutableY(0);
     auto &E = WS->mutableE(0);
+
+    double x(0);
     for (size_t i = 0; i < Y.size(); ++i) {
-      const double x = (X[i] + X[i + 1]) / 2;
+      x = (X[i] + X[i + 1]) / 2;
```

Comment:
Remove this and just use `X[i]` when computing `Y[i]`, the `points()` call above does the job.